#!/usr/bin/env python
PACKAGE = "navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("initial_x",    double_t, 0, "initial location in x", 1.2, -10, 10)
gen.add("initial_y",    double_t, 0, "initial location in y", 6, -10, 10)
gen.add("initial_yaw",  double_t, 0, "initial rotation yaw", -1.5708, -3.1416, 3.1416)

exit(gen.generate(PACKAGE, "localization", "Localization"))
